### ISSUE TITLE
WB: RangeError invalid string length in invest stdErr callback

### ISSUE THREAD

**Issue by davemfish** [natcap/invest#1092]
A user reported this error. It shows up as an uncaught exception in the main electron process. The user's logs are filled with an excessive amount of stderr from `sdr`. The following is logged over and over:

```python
Traceback (most recent call last):
  File "src\natcap\invest\sdr\sdr_core.pyx", line 293, in natcap.invest.sdr.sdr_core._ManagedRaster._load_block
AttributeError: 'NoneType' object has no attribute 'astype'
Exception ignored in: 'natcap.invest.sdr.sdr_core._ManagedRaster.get'
Traceback (most recent call last):
  File "src\natcap\invest\sdr\sdr_core.pyx", line 293, in natcap.invest.sdr.sdr_core._ManagedRaster._load_block
AttributeError: 'NoneType' object has no attribute 'astype'
```

Since we are naively concatenating all invest stderr to a single string and then passing it to the browser when invest exits, we are apparently exceeding node's limit on string length https://github.com/nodejs/node/issues/35973

Our stderr callback:
```javascript
    const stdErrCallback = (data) => {
      logger.debug(`${data}`);
      investStdErr += `${data}`;
    };
    investRun.stderr.on('data', stdErrCallback);
```

**Comment by davemfish** [comment]
I think I'll bundle the fix for this into #1089

**Comment by davemfish** [comment]
It would also be nice to log these uncaught node exceptions. I think we can catch, log, and re-raise them with this listener https://nodejs.org/api/process.html#process_event_uncaughtexception
